packages/application-components/src/components/modal-pages/form-modal-page/form-modal-page.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import ModalPage from '../internals/modal-page';
+import ModalPageTopBar from '../internals/modal-page-top-bar';
+import ModalPageHeader from '../internals/modal-page-header';
+import { ContentWrapper } from '../internals/modal-page.styles';
+import messages from '../internals/messages';
+
+const useLabel = (label, intl) =>
+  typeof label === 'string' ? label : intl.formatMessage(label);
```

Comment:
Wait, isn't this wrong? The prop-type says it's either a string or a node, not an intl message 🤔

Reply:
Hmm, this is used for the props `labelSecondaryButton`, `labelPrimaryButton`, which are either `string` or an `intl` shape. Aren't you confusing it with `subtitle`?

Reply:
You're right, I got confused 🤦‍♂️

We use the same in `dialog-footer.js`, so maybe we can rename it there as well.